### ISSUE TITLE
Yojee - Senior Frontend/Fullstack Engineer - Saigon, District 4 - FT/Hybrid

### ISSUE THREAD

**Issue by Lilyyojee** [awesome-jobs/vietnam#3593]
## Location

* District 4, Saigon, Vietnam

## Salary 

* Negotiate

## About the role

Yojee is looking for a Fullstack Engineer who enjoys developing features end-to-end in a cloud platform environment with React, Angular, React Native and Elixir/Phoenix. Due to our strong commitment to team ownership, you will have many opportunities to bring in your ideas and architecture suggestions. Since we are aware that you might be coming either from frontend development or backend development, missing experience on one side can be balanced with strong skills on the other end - but we expect the will to ramp up on the full stack.

## Requirements

* Minimum Bachelor Degree in Computer Engineering or its equivalent related experience.
* 2+ years of experience in development of various complex systems
* Experienced and proficient in working with at least two languages, preferably Elixir, Javascript or any other functional programming language.
* Experience with frontend technologies like Angular.js, React.js or vue.js
* Experience working with RDBMS (preferably PostgreSQL)
* Experience with React Native
* Good English communication. 

### Nice to have

* DevOps experience with AWS or Azure, Docker.
* Production experience with Elixir & Phoenix.
* Scripting (Python, Lambda, Bash).

## Responsibilities

* Review designs created by web designers; ask any necessary clarifying questions before starting on work
* Write code to implement those visual elements into various web applications
* Create prototypes, mockups, and staging sites for review and feedback
* Optimize websites and applications to ensure fast loading speeds
* Maintain websites and implement any necessary additions or enhancements
* Identify and troubleshoot various website issues and coding problems
* Collaborate with members of the team (including designers and product managers) to consistently improve functionality and user-friendliness of the web applications
* Maintain updated knowledge of the development industry and any advancements in technology

### Reporting Structure - Who Will Help You Win

* This role will be located in Vietnam and will report to EVP Technology & Innovation.

## Why it would be awesome to work with us

We are not a startup although the company structure is organized in such a way. Yojee is a public listed company on Australian stock exchange (ASX: YOJ). We are growing our team in Vietnam which is developing our core product. Don't miss this opportunity to join us early.

* Performance bonus: equivalent to 1-3months salary
* Wellness Benefits: we contribute AUD$500 annually to put towards anything that contributes to your mental or physical wellbeing.
* Learning and Development: We develop tailored development paths (because we are all unique) and contribute AUD $1,000 annually to invest in your continued education.
* Provided Macbook pro
* Occasional onsite trip to Singapore, Indonesia
* International environment : Australia, New Zealand, US, Singapore, India, Indonesia, Malaysia, etc,..
* Fun and regular company events to celebrate and connect with teammates
* Free coffee, milk and tea
* Free parking
* No 13th month salary
* Unlimited paid day off
* 100% official salary in probation time
* Full social insurance paid based on salary
* A positive environment building towards an industry that is 12% of Global GDP.
* Purpose: Supply Chains are processes and there are incredible opportunities to create new efficiencies which help businesses and the planet.
* Autonomy, empowerment and flexibility to deliver your best work.
* Cool tech (of course!).
* Growth opportunities via our You at Yojee (YaY) Program.

## About Yojee

Yojee Limited (ASX: YOJ) is a cloud based (SaaS) logistics platform that seamlessly and uniquely manages, tracks and optimizes freight movements along the entire logistics chain, from sender to end customer, across borders and between logistics providers (land, sea, air), with subcontractors and for multi-leg journeys. Rarely is a single carrier servicing an entire goods journey from sender to end customer, or exclusively using one type of transportation method. Yojee ensures connectivity and more efficient planning along the entire journey.

### About the Engineering team at Yojee

The backbone of Yojee focused on building the core platform and product using best in class technology. We love clean code, well-designed architecture and applying best practices.  We build cool technology that lasts long and scales well.

### How to apply

Please apply [here ](https://yojee.bamboohr.com/careers/149?source=aWQ9MjI%3D) if you are interested in. 

### Contact
Homepage: Yojee.com
Lily: lily@yojee.com
Phone: 0924605299
